KlingonAssistant/data/mem-07-l.xml

```diff
@@ -1269,7 +1278,7 @@ This word can be used to refer to traveling with or without a special purpose. I
       <column name="hidden_notes"></column>
       <column name="components"></column>
       <column name="examples">{wa'leS:n}, {cha'leS:n}</column>
-      <column name="examples_de"></column>
+      <column name="examples_de">{wa'leS:n}, {cha'leS:n}</column>
```

Comment:
When the examples_de is identical to examples, it's actually not necessary to repeat it.